### PR TITLE
Adding time delta to the 'resolved' comment 

### PR DESCRIPTION
**Pull request by two-trick-pony-NL**
Hi, 

**What does it do:**
After having a few downtime incidents I wondered how long it took for the downtime to resolve itself.  Thats why I figured to add the duration of the incident to the comment that Upptime posts once a website comes back online. 

I did it quite hacky but that is mainly because I could not find where functions go in the repo. Please let me know if: 
- You'd like to have that feature added then I'll clean it up, and make sure to test properly
- Where you'd like these kind of generic helper functions (finding a timedelta between 2 timestamps)

### CODE REVIEW

File: src/update.ts [hunk 1]
```diff
@@ -402,17 +422,20 @@ generator: Upptime <https://github.com/upptime/upptime>
...
                   ? "performance has improved"
                   : "is back up"
-                } in [\`${lastCommitSha.substr(
-                  0,
-                  7
-                )}\`](https://github.com/${owner}/${repo}/commit/${lastCommitSha}).`,
+              } in [\`${lastCommitSha.substr(
+                0,
+                7
+              )}\`](https://github.com/${owner}/${repo}/commit/${lastCommitSha}).\n\nTime taken: ${getHumanReadableTimeDifference(
```
Comment (AnandChowdhary) [inline src/update.ts:436]:
```suggestion
              )}\`](https://github.com/${owner}/${repo}/commit/${lastCommitSha}) after ${getHumanReadableTimeDifference(
```

**Review by AnandChowdhary — CHANGES_REQUESTED** [review]
Just changed the text a bit

**Review by two-trick-pony-NL — COMMENTED** [review]
Sure thing go

**Comment by AnandChowdhary** [comment]
Thank you!

**Pull request merged**